pkg/util/graph/graphsdk/applications/application_item_request_builder.go

```diff
@@ -0,0 +1,186 @@
+package applications
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f "github.com/microsoft/kiota-abstractions-go"
```

Comment:
Is there any way to make this generated API more readable?

Reply:
not really 🥲

Reply:
This is generated, right?